### DISCUSSION TITLE
Bricklink xml import - Which columns/fields can be imported

### DISCUSSION THREAD

**Discussion by mevitsbricks** [rgriebl/brickstore#439] [Q&A]
Do you have a list of the columns/fields that can be imported?  I tried changing a wanted list to Comments or MargerText but it wouldn't import those into brickstore.

Thanks,

Kevin

**Comment by ZZJHONS** [comment]
Maybe this help:

https://github.com/rgriebl/brickstore/blob/master/BrickStoreXML.rnc

**Reply by mevitsbricks** [reply]
Thanks, I looked through this one and it's how to check a bsx file.  I have attempted several of the tags in here on the XML import and it doesn't take them.
